protocol-definitions/MC.yaml

```diff
@@ -159,6 +159,12 @@ methods:
           since: 2.4
           doc: |
             Global indexes of the map.
+        - name: wanReplicationRef
+          type: WanReplicationRef
+          nullable: true
+          since: 2.7
+          doc: |
+            WanReplicationRef of the map.
```

Comment:
The documentation can be improved. It will show up at the javadoc, hence I suggest better explanation about what is WanReplicationRef as well.